src/components/encoding-pane/property-editor-schema.ts

```diff
@@ -181,9 +350,41 @@ export function generateFormData(shelfId: ShelfId, fieldDef: ShelfFieldDef) {
       const prop = customProp.prop;
       const nestedProp = customProp.nestedProp;
       const propertyKey = nestedProp ? prop + '_' + nestedProp : prop;
-      formData[propertyKey] = fieldDef[prop] ? nestedProp ? fieldDef[prop][nestedProp] : fieldDef[prop] : undefined;
+      const fData = fieldDef[prop] ? nestedProp ? fieldDef[prop][nestedProp] : fieldDef[prop] : undefined;
+      // Display empty string when '?' is passed in to retrieve auto generated visualization
```

Comment:
auto-generated visualization?    Do you mean default value.  

Also, when would '?' be passed in?  Can you explain more?